include/deal2lkit/parsed_grid_generator.h

```diff
@@ -77,6 +77,45 @@ class ParsedGridGenerator : public ParameterAcceptor
    */
   Triangulation<dim, spacedim> *serial();
 
+  /**
+   * Generate the grid. It is implemented is such a way there are no error due
+   * to the particual choice of dim and spacedim (some grid requires
```

Comment:
Can you copy-paste this documentation in the long description of the grid type? This should appear in the parameter file with long description as well.